### DISCUSSION TITLE
Agent gets disconnected from server

### DISCUSSION THREAD

**Discussion by web2wire** [pyToshka/docker-wazuh-agent#140] [Q&A]
Hi,

I have a version of this repo working with some tweaks for maintaining persistence in a mounted volume (more fiddly than I expected). The container starts up, runs register_agent.py as expected, the agent shows up in the Wazuh manager and all looks good. However after a few minutes it goes from 'active' to 'disconnected' and it appears that the agent is no longer talking to the manager and vice versa, though neither report any specific errors.  This is different from when agents are installed from normal packages which seem to remain active the whole time. I think in my current case both server and client are just listening for the other to do something and the manager is expecting some sort of keepalive.

I don't know if this is expected behaviour or not but having several agents all sitting disconnected and never seeming to wake up doesn't feel like it's behaving correctly.

Does anyone have any insights into whether this is normal or any way or working around it?

Thanks

**Comment by zakkg3** [comment]
Same behaviour here. 
I belive we should open this disssion as for the upvotes it seems something its not working.
